config/settings.yml

```diff
@@ -1069,6 +1070,7 @@
   :worker_base:
     :defaults:
       :count: 1
+      :cpu_threshold: 500
```

Comment:
@carbonin what do you think of keeping this as a percentage and converting to millicore in container_common?  I think cpu percentage is more intuitive to people